docusaurus-docs/conda-store-ui/tutorials/create-envs.md

```diff
@@ -83,6 +83,21 @@ To install packages published only on [PyPI][pypi] using [`pip`][pip], include a
 
 -->
 
+### Set environment variables
+
+:::note
+This features is in the release *after* conda-store version 2024.1.1.
+
+Currently, only the `CONDA_OVERRIDE_CUDA` environment variable is supported to set the CUDA version for GPU-powered workflows.
+:::
+
+You can set environment variables in the YAML-editor with the following syntax:
+
+```yaml
+variables:
+   CONDA_OVERRIDE_CUDA: 12
```

Comment:
```suggestion
    CONDA_OVERRIDE_CUDA: '12.0'
```

I suggest making this change because it makes it clear that the value is a string.